tests/test_widerface_format.py

```diff
@@ -398,8 +403,50 @@ def test_inplace_save_writes_only_updated_data(self):
                 ignored_attrs=IGNORE_ALL,
             )
 
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_can_convert_to_widerface(self):
+        source_dataset = Dataset.import_from(MANGLING_DATASET_DIR, "wider_face")
+
+        with TestDir() as test_dir:
+            source_dataset.export(test_dir, "wider_face", save_media=True)
+            parsed_dataset = Dataset.import_from(test_dir, "wider_face")
+
+            compare_datasets(self, source_dataset, parsed_dataset, require_media=True)
+
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_can_convert_to_pascal_voc(self):
+        source_dataset = Dataset.import_from(MANGLING_DATASET_DIR, "wider_face")
+
+        with TestDir() as test_dir:
+            source_dataset.export(test_dir, "voc", save_media=True)
+            parsed_dataset = Dataset.import_from(test_dir, "voc")
+            parsed_dataset.transform("remap_labels", mapping={"background": ""})
+
+            compare_datasets(
+                self,
+                source_dataset,
+                parsed_dataset,
+                require_media=True,
+                ignored_attrs={"difficult", "truncated", "occluded"},
+                externally_comparison=True,
+            )
+
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_can_filter_by_subsets(self):
+        source_dataset = Dataset.import_from(MANGLING_DATASET_DIR, "wider_face")
+
+        first_dataset = source_dataset.filter("/item/annotation[label='face']")
+        second_dataset = source_dataset.filter("/item/annotation[label!='face']")
+
+        merger = IntersectMerge()
+        merged_dataset = merger([first_dataset, second_dataset])
+
+        compare_datasets(
+            self, source_dataset, merged_dataset, require_media=True, ignored_attrs={"score"}
+        )
```

Comment:
This test doesn't seem to be aligned with its name. Why the merging is being done? Why the subsets are obtained by label filtering?